app/src/components/blocks/lloydGeorgeFileInputStage/LloydGeorgeFileInputStage.tsx

```diff
@@ -40,15 +42,22 @@ function LloydGeorgeFileInputStage({ documents, setDocuments, setStage }: Props)
         ? getFormattedDate(new Date(patientDetails.birthDate))
         : '';
     let fileInputRef = useRef<HTMLInputElement | null>(null);
-
-    const { handleSubmit, control, formState } = useForm();
-    const lgController = useController(lloydGeorgeFormConfig(control));
-
-    const hasFileInput = documents.length;
+    const [uploadFilesErrors, setUploadFilesErrors] = useState<Array<UploadFilesErrors>>([]);
+    const hasFileInput = !!documents.length;
+    const [showNoFilesMessage, setShowNoFilesMessage] = useState<boolean>(false);
     const baseUrl = useBaseAPIUrl();
     const baseHeaders = useBaseAPIHeaders();
-
     const uploadDocuments = async () => {
+        setShowNoFilesMessage(!hasFileInput);
+        if (!hasFileInput) {
+            window.scrollTo(0, 0);
+            return;
+        }
+        setUploadFilesErrors(uploadDocumentValidation(documents));
+        if (!!uploadFilesErrors.length) {
```

Comment:
```suggestion
        if (uploadFilesErrors.length) {
```
I think this evaluates to true or false already

Reply:
uploadFilesErrors.length return a number

Reply:
the if statement converts the condition to a boolean already